app/react/portainer/account/CreateAccessTokenView/CreateUserAccessToken.test.tsx

```diff
@@ -0,0 +1,57 @@
+import userEvent from '@testing-library/user-event';
+
+import { act, renderWithQueryClient } from '@/react-tools/test-utils';
+import { UserViewModel } from '@/portainer/models/user';
+import { UserContext } from '@/react/hooks/useUser';
+
+import { CreateUserAccessToken } from './CreateUserAccessToken';
+
+test('the button is disabled when description is missing and enabled when description is filled', async () => {
+  const queries = renderComponent();
+
+  const button = queries.getByRole('button', { name: 'Add access token' });
+  expect(button).toBeDisabled();
+
+  const descriptionField = queries.getByLabelText(/Description/);
+
+  await act(() => userEvent.type(descriptionField, 'description'));
+  expect(button).toBeEnabled();
+
+  userEvent.type(descriptionField, 'description');
+  expect(button).toBeEnabled();
+
+  // act(() => userEvent.clear(descriptionField));
+  // // await act(() => Promise.resolve()); // Flush microtasks used by Formik validation
+
+  // expect(button).toBeDisabled();
+});
+
+// test('once the button is clicked, the access token is generated and displayed', async () => {
```

Comment:
let's either remove them or uncomment and change `test(...)` to `test.skip(....)`